src/plonk.rs

```diff
@@ -363,20 +391,27 @@ impl<C: HaloCurve> Circuit<C> {
 
         debug_assert_eq!(halo_a.len(), 1);
         debug_assert_eq!(halo_b.len(), 1);
-        let schnorr_proof = self.schnorr_protocol(halo_a[0], halo_b[0], halo_g, randomness, u_curve, &mut challenger);
+        let schnorr_proof = self.schnorr_protocol(
+            halo_a[0],
+            halo_b[0],
+            halo_g,
+            randomness,
+            u_curve,
+            &mut challenger,
+        );
 
         Ok(Proof {
-            c_wires,
-            c_plonk_z,
-            c_plonk_t,
+            c_wires: c_wires.iter().map(|c| c.to_affine()).collect(),
```

Comment:
Could also use `ProjectivePoint::batch_to_affine` (and for `c_plonk_t` below)

Reply:
Actually in this case `c_wires` and `c_plonk_t` are  `PolynomialCommitment` structs, and I assume that `PolynomialCommitment::batch_to_affine` has been run on them previously, so the `to_affine` is basically costless.
The `PolynomialCommitment` is a bit wonky so I can change it if you want.

Reply:
Oh I see, looks reasonable then.